CHANGELOG.md

```diff
@@ -20,7 +20,7 @@ Release Versions
 
 - fix(robot-model): improve ik performance (#205)
 - fix(robot-model): kinematics tests (#207)
-- feat: add integrate and differentiate methods to Cartesian types (#209)
+- feat: add integrate and differentiate methods to Cartesian and joint types (#193)
```

Comment:
```suggestion
- feat: add integrate and differentiate methods to Cartesian and joint types (#193,#209)
```
right?

Reply:
Well the parent issue is 193, hence I only put that. PRs would be 209 and 210 but they are both linked to the parent issue